coderd/database/migrations/000066_app_display_name.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
```

Comment:
Is there a purpose to having these in separate migrations? I think we'll need to bump these too

Reply:
not really that's just the way they were made, they're kinda large though so I'm just gonna keep them as separate files